### ISSUE TITLE
Editor text legibility in Edge

### ISSUE THREAD

**Issue by kevinansfield** [TryGhost/Ghost#8867]
### Issue Summary

In MS Edge on Windows 10 the editor font rendering makes it quite difficult to read.

**I'm not sure if this is an artefact of running in a VM or if this is a real issue, would be great if anyone with a real windows machine could confirm!**

Edge (looks worse on my Dell than on my MBP screen):
<img width="1164" alt="screen shot 2017-08-10 at 09 56 17" src="https://user-images.githubusercontent.com/415/29162844-863791a6-7db2-11e7-9085-dc865ef951b3.png">

Chrome/Windows:
<img width="1164" alt="screen shot 2017-08-10 at 09 56 43" src="https://user-images.githubusercontent.com/415/29162878-9eaaf476-7db2-11e7-91c3-292a77564893.png">

### Technical details:

* Ghost Version: 1.5.1
* Browser/OS: Edge/Windows 10

**Comment by vickychijwani** [comment]
I have a Windows dual-boot so I tested this out, it is a real issue.

And I think I've found the root cause as well: the double quotes around `Consolas` in the CSS [here](https://github.com/TryGhost/Ghost-Admin/blob/eec3714c2b7415dddfe43a6f69d5d50de7b02c66/app/styles/layouts/editor.css#L480) are "smart" curly quotes instead of straight "dumb" ones (`“”` instead of `""`). Apparently Edge cannot parse the smart quotes and so it falls back to the default monospace font which is the ugly Courier New on Windows. It's pretty great that Chromes handles smart quotes gracefully.

Confirmed by editing the CSS through devtools - I just deleted the offending characters and inserted normal quotes, and it worked 😅
